### DISCUSSION TITLE
Optimization to reduce loads

### DISCUSSION THREAD

**Discussion by tilucas97** [NatLabRockies/floris#702] [Q&A] [answered]
Hello everyone,

I hope you're doing well. I have a question regarding the optimization of wind farms using Floris. I am curious to know if Floris can be utilized to optimize wind farms with the primary objective of reducing loads, specifically through yaw optimization, rather than focusing solely on power maximization.

I appreciate any insights or information you can provide on this matter. Thank you in advance for your assistance.

Best regards,
Tiago Frutuoso

**Comment by Bartdoekemeijer** [comment, marked as answer]
Hi Tiago,

Coupling simple engineering wake models with loads models has been done in the literature, but is not currently a feature in FLORIS. I don't have an exhaustive list for you, but here are a couple of relevant articles:
1. Here's an article in which FLORIS is coupled to CCBlade (loads solver) to do combined optimizations between power production and loads. This nicely demonstrates the concept. See the article here: https://www.sciencedirect.com/science/article/pii/S0360544217300518.
1. ECN.TNO has coupled their engineering wake model called Farmflow with their in-house aero-elastic turbine loads simulation software. To make the optimization computationally feasible, a large number of aero-elastic simulations are performed and a lookup table is generated from which then is interpolated in real-time. The loads look-up table is validated in this article: https://wes.copernicus.org/articles/4/549/2019/wes-4-549-2019.pdf.
2. Here's another article on surrogate loads modeling by my current-colleague and ex-NREL researcher Kelsey Shaler: https://iopscience.iop.org/article/10.1088/1742-6596/2265/3/032095.

More people in academia and industry have built surrogate loads models to then perform combined loads and power optimizations. However, questions remain surrounding the reliability of those loads models. Namely, engineering wake models like FLORIS model 10-minute-averaged flow properties, while structural loads require simulations sampled in the order of milliseconds to seconds.

Cheers,
Bart

**Reply by tilucas97** [reply]
Thank you very much!
